src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/SchedulerTab.php

```diff
@@ -123,37 +127,152 @@ private function renderRunner(): void
-        $instructionsHTML = '';
+        $instructions = [];
         if (CronRegistrar::RUNNER_WPCRON_CRONTAB === $runner) {
             $url = site_url('wp-cron.php');
             $instructions = [
-                __('Make sure `DISABLE_WP_CRON` AND `ALTERNATE_WP_CRON` is set to false.', I18N::DOMAIN),
                 __('Check for admin notices related to cron.', I18N::DOMAIN),
-                sprintf(__('Add `*/1 * * * * curl %s` to crontab', I18N::DOMAIN), $url),
+                __('Check if `curl` and `cron` is installed in the website\'s server.', I18N::DOMAIN),
+                sprintf(__('Add %s to crontab.', I18N::DOMAIN), "<code>* * * * * curl $url</code>"),
                 __('Upon changing runner, please make sure to remove the cron above from crontab.', I18N::DOMAIN),
             ];
         } elseif (CronRegistrar::RUNNER_CRONTAB_API === $runner) {
             $url = rest_url(EndpointLoader::getFullNamespace().'/'.TaskProcessorEndpoint::ROUTE);
             $instructions = [
-                __('Make sure `DISABLE_WP_CRON` is set to true.', I18N::DOMAIN),
                 __('Check if `curl` and `cron` is installed in the website\'s server.', I18N::DOMAIN),
-                sprintf(__('Add `*/1 * * * * curl %s` to crontab', I18N::DOMAIN), $url),
+                sprintf(__('Add %s to crontab.', I18N::DOMAIN), "<code>* * * * * curl $url</code>"),
                 __('Upon changing runner, please make sure to remove the cron above from crontab.', I18N::DOMAIN),
             ];
+        }
+
+        if (!empty($instructions)) {
+            $instructionsHTML = '';
+            foreach ($instructions as $key => $instruction) {
+                $instructionsHTML .= '<p style="white-space: pre-line;">'.($key + 1).'. '.$instruction.'</p>';
+            }
+
+            echo $this->getFormGroup(
+                __('Instructions', I18N::DOMAIN),
+                $instructionsHTML
+            );
+        }
+    }
+
+    private function renderRequirements(): void
+    {
+        echo '<br>'.$this->getFormHeader(__('Requirements', I18N::DOMAIN));
+
+        $runner = StoreKeeperOptions::get(StoreKeeperOptions::CRON_RUNNER, CronRegistrar::RUNNER_WPCRON);
+
+        $disableWpCron = false;
+        if (defined('DISABLE_WP_CRON')) {
+            $disableWpCron = DISABLE_WP_CRON;
+        }
+
+        $alternateWpCron = false;
+        if (defined('ALTERNATE_WP_CRON')) {
+            $alternateWpCron = ALTERNATE_WP_CRON;
+        }
+
+        $helpText = __('See documentation', I18N::DOMAIN);
+        if (CronRegistrar::RUNNER_WPCRON_CRONTAB === $runner) {
+            $hasHelp = !$disableWpCron || $alternateWpCron;
+            $data = [
+                [
+                    'key' => 'DISABLE_WP_CRON',
+                    'value' => $disableWpCron ? 'true' : 'false',
+                    'validity' => $this->generateValidityContent($disableWpCron),
```

Comment:
```suggestion
                    'validity' => $this->generateExpectValidityContent(false,$disableWpCron),
```
That would be way easier to understand

Reply:
Removed this function.